.github/workflows/go_wasm_build_test_deploy.yml

```diff
@@ -25,13 +29,25 @@ jobs:
         go-version: '1.22.5'
```

Comment:
Convert all the versions (Go, OS, Browser etc) to GitHub action variables and refer to the environment variable instead. It'll be easier to change/revert them.